src/main/frontend/components/query_table.cljs

```diff
@@ -165,52 +167,66 @@
           ;; Sort state needs to be in sync between final result and sortable title
           ;; as user needs to know if there result is sorted
           sort-state (get-sort-state current-block)
-          result' (sort-result result sort-state)]
-      [:div.overflow-x-auto {:on-mouse-down (fn [e] (.stopPropagation e))
-                             :style {:width "100%"}
-                             :class (when-not page? "query-table")}
-       [:table.table-auto
-        [:thead
-         [:tr.cursor
-          (for [column columns]
-            (let [title (if (and (= column :clock-time) (integer? clock-time-total))
-                             (util/format "clock-time(total: %s)" (clock/seconds->days:hours:minutes:seconds
-                                                                   clock-time-total))
-                             (name column))]
-              (sortable-title title column sort-state (:block/uuid current-block))))]]
-        [:tbody
-         (for [row result']
-           (let [format (:block/format row)]
+          result' (sort-result result sort-state)
+          table-version (get-shui-component-version :table config)
+          result-as-text (for [result-entry result'] 
+                           (for [column-name columns] 
+                             (case column-name 
+                               :block (or (get-in result-entry [:block/page :block/original-name])
```

Comment:
`:block` and `:page` reversed?

Reply:
btw there's a bottom border for the table.
<img width="2451" alt="CleanShot 2023-04-03 at 22 38 23@2x" src="https://user-images.githubusercontent.com/479169/229543303-1a114b45-4313-4f55-8916-ce8270622185.png">

Reply:
This is a scrollbar, some work before launch needs to be done to remove it (it's visually unpleasant but for accessibility reasons we shouldn't just entirely disable it). I'd like to move that to an improvement before launching v2 as the default (should not block merge)